src/GuppyDataExplorer/ExplorerVisualization/index.jsx

```diff
@@ -59,6 +61,10 @@ class ExplorerVisualization extends React.Component {
     const chartData = this.getData(this.props.aggsData, this.props.chartConfig, this.props.filter);
     const tableColumns = (this.props.tableConfig.fields && this.props.tableConfig.fields.length > 0)
       ? this.props.tableConfig.fields : this.props.allFields;
+    const isComponentLocked = checkForAnySelectedUnaccessibleField(this.props.aggsData,
+      this.props.accessibleFieldObject, this.props.guppyConfig.accessibleValidationField);
+    const lockMessage = `You cannot see this chart because it contains items under ${this.props.tierAccessLimit} ${this.props.nodeCountTitle.toLowerCase()}`;
```

Comment:
I don't think the ${this.props.nodeCountTitle.toLowerCase()} accounts for plurals, "50 case" sounds awkward

Reply:
The lock message will be rephrased as suggested by @ZakirG here https://github.com/uc-cdis/gen3-ui-component/pull/39#discussion_r282515232

Reply:
also for plurals, how about we use this? https://github.com/blakeembrey/pluralize